pages.de/common/steamcmd.md

```diff
@@ -0,0 +1,16 @@
+# steamcmd
+
+> Ein Kommandozeilenwerkzeug, um über Steam verfügbare Anwendungen zu verwalten.
+> Weitere Informationen: <https://manned.org/steamcmd>.
+
+- Installiere und aktualisiere eine Anwendung ohne dich einzuloggen:
+
+`steamcmd +login anonymous +app_update {{anwendungs_id}} +quit`
```

Comment:
```suggestion
`steamcmd +login {{anonymous}} +app_update {{anwendungs_id}} +quit`
```
Translate `{{anonymous}}` too

Reply:
The string `anonymous` is a constant required for the anonymous login, thus we can't translate it. Maybe we also should change this in the English version of the page by removing the braces. See https://developer.valvesoftware.com/wiki/SteamCMD#Anonymous

Reply:
I guess we no need to translate `anonymous` as you suggested. Apply the curly braces in this suggestion alone and the PR is good to go.

Reply:
Thanks, I've applied your suggestion